packages/terra-time-input/src/TimeInput.module.scss

```diff
@@ -64,66 +92,77 @@
   }
 
   .time-input-group {
-    margin-left: var(--terra-time-input-time-input-group-margin-left, 0.357em);
-    margin-right: var(--terra-time-input-time-input-group-margin-right, 0.357em);
-
     &:first-of-type {
       margin-left: 0;
+      margin-right: var(--terra-time-input-time-input-group-hour-margin-right, 0.14285714rem);
     }
 
-    &:last-of-type {
+    &:nth-of-type(2) {
+      margin-left: var(--terra-time-input-time-input-group-minute-margin-left, 0.14285714rem);
       margin-right: 0;
     }
   }
 
   .time-input-hour,
   .time-input-minute {
-    background-color: var(--terra-time-input-background, #fff);
-    background-image: var(--terra-time-input-animation-background, none);
+    background-color: var(--terra-time-input-background-color, #fff);
+    background-image: var(--terra-time-input-background-image, none);
     background-position: 0 100%;
     background-repeat: no-repeat;
-    background-size: var(--terra-time-input-background-start, 0 0);
+    background-size: var(--terra-time-input-background-size, 0 0);
     border-color: var(--terra-time-input-border-color, #dedfe0);
     border-radius: var(--terra-time-input-border-radius, 3px);
     border-style: var(--terra-time-input-border-style, solid);
     border-width: var(--terra-time-input-border-width, 1px);
     box-sizing: border-box;
+    caret-color: var(--terra-time-input-caret-color, #1c1f21);
     color: var(--terra-time-input-color, #1c1f21);
     display: table;
-    height: var(--terra-time-input-mobile-height, 1.857rem);
-    min-width: var(--terra-time-input-width-input, 3.57em);
-    padding-bottom: var(--terra-time-input-input-padding-bottom, 0.357em);
-    padding-left: var(--terra-time-input-input-padding-left, 0.357em);
-    padding-right: var(--terra-time-input-input-padding-right, 0.357em);
-    padding-top: var(--terra-time-input-input-padding-top, 0.357em);
+    height: var(--terra-time-input-mobile-height, 2.1428571429rem);
```

Comment:
Let's remove the word, `mobile` from these variables

Reply:
If the variable without the `mobile` word already exists in this file, like `--terra-time-input-line-height`, let's update it to `--terra-time-input-desktop-line-height`.

Reply:
As per offline discussion, we are leaving this as is for now. So I am gonna mark this has resolved.